sdks/java/extensions/avro/build.gradle

```diff
@@ -35,7 +41,12 @@ def avroVersions = [
     '1111': "1.11.1",
 ]
 
-avroVersions.each { k, v -> configurations.create("avroVersion$k") }
+avroVersions.each { k, v ->
+  configurations {
+    create("avroVersion$k").extendsFrom(implementation)
```

Comment:
extends the implementation configuration.
`avroVersion$k` should not extends test as we do not want to include code generated by the `1.8.2` version used in test